lighthouse-core/report/html/report-styles.css

```diff
@@ -104,6 +104,10 @@
   --env-name-min-width: 220px;
   --env-tem-padding: 10px 0px;
   --header-padding: 20px 0 20px 0;
+  --plugin-badge-bg: var(--color-white);
+  --plugin-badge-size-big: calc(var(--circle-size-big) / 2.7);
```

Comment:
looks like you can drop the -big variant now :)

Reply:
hmm, can you take another look, b/c this seems necessary to me. the plugin badge is larger for the gauge in the section, so the `--plugin-badge-size-big` var is derived from the `--circle-size-big` var

Reply:
@paulirish ^

Reply:
ah true. 

what i was probably thinking was changing uses of circle-size-big to assign it to circle size:

```diff
     .lh-category .lh-gauge {
-      width: var(--circle-size-big);
-      height: var(--circle-size-big);
+      --circle-size: var(--circle-size-big);
     }
```

and then all your base stuff is derived off the same value. :D

Reply:
ok but i accidentally did it for #8222

Reply:
btw this specific example didnt work, i think there's a problem with the data dependency of the variables / scope.